pkg/gather/clusterconfig/workloads.go

```diff
@@ -34,6 +34,7 @@ import (
 const (
 	// workloadGatherPageSize is 500 (the default for Kube).
 	workloadGatherPageSize = 500
+	podsLimit              = 3000
```

Comment:
Explaining the "magic-number" might come in handy later

Reply:
Good point. Let me add some "explanation"